packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -1248,6 +1252,8 @@ describe('plugin-meetings', () => {
       });
       describe('#closeLocalStream', () => {
         it('should stop the stream, and trigger a media:stopped event when the local stream stops', async () => {
+          meeting.mediaProperties.audioTrack = {readyState: 'ended'};
+          meeting.mediaProperties.videoTrack = {readyState: 'ended'};
```

Comment:
Same, this is stubbed already but the beforeEach.

Reply:
@anthonywebb can you point me to the line , I could not find it and it was failing in master